src/components/image-viewer/ImageInfo.vue

```diff
@@ -23,18 +23,14 @@
         <label>{{ $gettext(document.image_type) }}</label>
       </p>
 
-      <h4 class="title is-4 has-text-light" v-translate>
-        Settings
-      </h4>
-
       <ul>
         <li v-if="document.camera_name"> {{ document.camera_name }}</li>
-        <li v-if="document.exposure_time" :title="$gettext('exposure_time')">{{ document.exposure_time }}s</li>
-        <li v-if="document.fnumber" :title="$gettext('fnumber')">f/{{ document.fnumber }}</li>
+        <li v-if="document.exposure_time" :title="$gettext('exposure_time')">1/{{ Math.floor(1/document.exposure_time) }}&nbsp;s</li>
+        <li v-if="document.fnumber" :title="$gettext('fnumber')">f/{{ Math.round(document.fnumber * 10) / 10 }}</li>
         <li v-if="document.focal_length" :title="$gettext('focal_length')">{{ document.focal_length }}&nbsp;mm</li>
-        <li v-if="document.iso_speed" :title="$gettext('iso_speed')">{{ document.iso_speed }} ISO</li>
+        <li v-if="document.iso_speed" :title="$gettext('iso_speed')">{{ document.iso_speed }}&nbsp;ISO</li>
         <li v-if="document.width && document.height" :title="$gettext('resolution')">
-          {{ document.height }} x {{ document.width }} <span translate>pixels</span>
+          {{ document.height }} x {{ document.width }}&nbsp;<span translate>pixels</span>
```

Comment:
What's the point of translating "pixels"?

Reply:
Side not, it's a typo : the good directive is `v-translate`. I guess i've copied the template from V6 without suffisant attention :) 

That said, "pixel" is not universal, I agree with V6 template, it should be translated.